inference-engine/src/gna_plugin/gna_infer_request.hpp

```diff
@@ -82,11 +82,12 @@ class GNAInferRequest : public InferenceEngine::AsyncInferRequestInternal {
         if (_callback) {
             auto infer_request = _publicInterface.lock();
             IE_ASSERT(infer_request != nullptr);
-            auto res = Wait(0);
+            auto res = Wait(InferenceEngine::IInferRequest::WaitMode::RESULT_READY);
             _callback(infer_request, res);
         }
     }
 
+
```

Comment:
Actually this blank line is not needed.

Reply:
Strange. Will change when merging into the master.